src/Compilers/Core/CodeAnalysisTest/CorLibTypesTests.cs

```diff
@@ -192,10 +192,18 @@ public void ConstantValueGetHashCodeTest02()
         public void ConstantValueToStringTest01()
         {
             var cv = ConstantValue.Create(null, ConstantValueTypeDiscriminator.Null);
+#if NETCOREAPP
```

Comment:
Since this is specifically a ConstantValue ToString test I guess we can keep testing the string it produces, but maybe we can extract the value `Null` or `Nothing` to a variable to avoid multiple `#if`s